BlazarUI/app/scripts/components/sidebar/BuildsSidebarContainer.jsx

```diff
@@ -56,17 +57,30 @@ class BuildsSidebarContainer extends Component {
   }
 
   render() {
+    let miniLogo;
+    if (this.props.isCollapsed) {
+      miniLogo = <SidebarLogo mini={true} />
+    }
     // BuildsNotifier.updateModules(this.state.builds.modules);
     return (
-      <Sidebar>
-        <BuildsSidebar
-          builds={this.state.builds}
-          stars={this.state.stars}
-          loading={this.state.loading} 
-          persistStarChange={this.persistStarChange} />
-      </Sidebar>
+      <div>
+        {miniLogo}
```

Comment:
Is there a reason we need to return null? I know I've read you can/should return null for an empty component, but I haven't seen anything that said you can return undefined. That said, it appear to work returning undefined? 

```javascript
renderMiniLogo() {
  if (this.props.isCollpased) {
    return (
      <SidebarLogo mini={true} />
    );
}
```